src/sentry/incidents/endpoints/organization_incident_index.py

```diff
@@ -38,12 +38,12 @@ def get(self, request, organization):
         query_start = request.GET.get("start")
         if query_start is not None:
             # exclude incidents closed before the window
-            incidents = incidents.exclude(date_closed__lte=query_start)
+            incidents = incidents.exclude(date_closed__lt=query_start)
```

Comment:
It doesn't matter much, but isn't using `exclude(..._lt)` the same thing as `include(...gte)`? I feel like the inversion makes it harder to parse

Reply:
I think we need to exclude based on either of the filters, not sure inverting like that would work or would I need to change the logic a bit but I can refactor this to make it more readable later